ui/narrative/methods/align_reads_using_tophat2/display.yaml

```diff
@@ -107,9 +107,10 @@ parameters :
             Set the condition associated with the input reads object. This is required for a single sample, but ignored for sets of samples, since that is included in the set.
 description : |
     <p>This App aligns the sequencing reads for a set of two or more samples to a genome using TopHat2 in order to identify splice junctions between exons with the help of Bowtie2 mapping program. Alternatively, if the user has only a single sample of reads or set of reads, this App will take single-end or paired-end library object or reads set instead of a sample set.</p>
+     <p>In addition, it outputs the Qualimap generated BAM alignment QC report that includes global summary such as total number of mapped reads, mean samples coverage, mean samples GC-content, mean samples mapping quality, and sample statistics for each sample along with PCA and coverage statistics in histograms.</p>
     <p><strong>NOTE:</strong> This method is one of the steps of the <a href= "http://kbase.us/rna-seq/"> KBase RNA-seq Pipeline </a>, however it can also be run standalone for one or more samples.</p>
   
-    <p><strong>Team members who developed & deployed algorithm in KBase:</strong> Tianhao Gu</p>
+    <p><strong>Team members who developed & deployed algorithm in KBase:</strong> Srividya Ramakrishnan, Sunita Kumari, Shinjae Yoo, Priya Ranjan, Jim Thomason, Vivek Kumar, Tianhao Gu</p>
```

Comment:
I don't know who Srividya is. If you want to put production team members, put people who help testing. I don't think others contribute to this repo at all.

Reply:
@r2sunita regardless. whatever...

Reply:
@Tianhao-Gu  Yes, agree. we need to include more names in the list. It needs to be done in all RNA-seq apps. After RSV, we can update author list. Thanks.
Sunita